README.md

```diff
@@ -6,23 +6,17 @@
 
 Heat is a distributed tensor framework for high performance data analytics.
 
-Project Status
---------------
+# Project Status
+
+[![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.2531472.svg)](https://doi.org/10.5281/zenodo.2531472)
 [![Mirror and run GitLab CI](https://github.com/helmholtz-analytics/heat/actions/workflows/ci_cb.yml/badge.svg)](https://github.com/helmholtz-analytics/heat/actions/workflows/ci_cb.yml)
 [![Documentation Status](https://readthedocs.org/projects/heat/badge/?version=latest)](https://heat.readthedocs.io/en/latest/?badge=latest)
 [![codecov](https://codecov.io/gh/helmholtz-analytics/heat/branch/main/graph/badge.svg)](https://codecov.io/gh/helmholtz-analytics/heat)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 [![license: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://opensource.org/licenses/MIT)
 [![Downloads](https://pepy.tech/badge/heat)](https://pepy.tech/project/heat)
 
-NEW!
---------------
-- [Quick Start](quick_start.md) for new users and contributors (Jan 14, 2023)
-
-
```

Comment:
I'd like to keep the NEW section so that users can see at a glance if there are any news.

Reply:
Sure, That would be great! to make any sort of announcements.👍

Reply:
@ClaudiaComito Done...👍. I added the NEW Section at the top, below the project status, as it will be used to announce any news related to the project, if you want me to change the position please tell me.